deepmd/entrypoints/compress.py

```diff
@@ -121,6 +122,7 @@ def compress(
             INPUT=control_file,
             init_model=None,
             restart=None,
+            init_frz_model=None,
```

Comment:
In compress.py why 'stage 3:transfer' still remains?

Reply:
Another PR will address this problem, this PR mainly focus on the model compression training support.